keyboards/rocketboard_16/ld/STM32F103C8T6_custom.ld

```diff
@@ -0,0 +1,85 @@
+/*
```

Comment:
No need for this custom ldscript anymore: https://github.com/qmk/qmk_firmware/tree/master/platforms/chibios/boards/STM32_F103_STM32DUINO/ld
The rules.mk stuff will have to stay though, until the issue sigprof mentioned is resolved.